BSB_lan.ino

```diff
@@ -441,15 +441,36 @@
 #include "BSB_lan_config.h"
 #include "BSB_lan_defs.h"
 
+#if defined(__AVR__)
 #include <avr/pgmspace.h>
-//#include <avr/wdt.h>
+#include <EEPROM.h>
+#endif
 #include <Arduino.h>
+
+#if defined(__AVR__) || defined(__arm__)
 #include <SPI.h>
- //Mega
-#if defined(__AVR__)
+#include <Wire.h>
+#endif
+
+#if defined(ESP32)
 #include <EEPROM.h>
+#include <ESPmDNS.h>
+EEPROMClass EEPROM("eeprom1", 0x800);
+#define strcpy_PF strcpy
+#define strcat_PF strcat
+#define strchr_P strchr
+#if defined(EEPROM_ERASING_PIN)
+#undef EEPROM_ERASING_PIN
+#define EEPROM_ERASING_PIN 34
+#endif
+#if defined(EEPROM_ERASING_GND_PIN)
+#undef EEPROM_ERASING_GND_PIN
+#endif
+#ifndef LED_BUILTIN 
+#define LED_BUILTIN 2
+#endif
+
 #else // __SAM3X8E__
```

Comment:
It seems that this else comes from the `#if defined(ESP32)` above - that would mean that this `else` is executed for both ARM and AVR platforms. AVRs, however do not need I2C_EEPROM, but the standard EEPROM library.

Reply:
> Ah, so this is meant to be a solution to get the whole BSB-LAN program running on that ESP32-Olimex-board or even on any ESP32 board (cuz not everybody needs POE)? That could be cool!
> Did you also already check if the SD card functionality works as expected with that Olimex board?

SD card support is still on my list.

Reply:
This comment was probably meant for a general reply. Is the inclusion of the correct EEPROM library fixed?

Reply:
Sorry, wrong comment. My fault.

Yes, inclusion of correct EEPROM library should be fixed now.